pwiz_tools/Skyline/Model/PeptideGroup.cs

```diff
@@ -332,11 +332,12 @@ public bool Equals(FastaSequence obj)
         {
             if (ReferenceEquals(null, obj)) return false;
             if (ReferenceEquals(this, obj)) return true;
-            return Equals(obj._name, _name) &&
-                Equals(obj._description, _description) &&
-                Equals(obj._sequence, _sequence) &&
-                ArrayUtil.EqualsDeep(obj.Alternatives, Alternatives) &&
-                obj.IsDecoy == IsDecoy;
+            var equal = Equals(obj._name, _name) &&

```

Comment:
if this is just a formatting change, it should be backed out (along with any similar changes)